bigtable-client-core/src/main/java/com/google/protobuf/BigtableZeroCopyByteString.java

```diff
@@ -8,10 +8,10 @@
  * of the objects de-serialized from the wire (which already do one copy, on
  * top of the copies the JVM does to go from kernel buffer to C buffer and
  * from C buffer to JVM buffer).
- *
- * @since 0.96.1
  */
 public final class BigtableZeroCopyByteString extends LiteralByteString {
```

Comment:
You're right.  It doesn't.  I copied the code from HBase, and I didn't pay too much attention.  Good catch.